src/main/java/com/moneylendingapp/security/jwt/JwtConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.moneylendingapp.security.jwt;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "application.jwt")
+@Getter
+@Setter
+@NoArgsConstructor
+public class JwtConfig {
+
+    private String secretKey;
+    private int tokenExpirationAfterDays;
```

Comment:
why not use Duration instead of int. expiryDuration is fine as the config name

Reply:
etExpiration(java.sql.Date.valueOf(LocalDate.now()
                        .plusDays(jwtConfig.getExpiryDuration())))

While setting the expiration days in the createToken method, the plusDays only takes a type of long